src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -47,6 +51,14 @@ public void parseCommand_addContact() throws Exception {
         assertEquals(new AddCommand(person), command);
     }
 
+    @Test
+    public void parseCommand_addTodo() throws Exception {
+        Todo todo = new TodoBuilder().build();
+        AddTodoCommand command = (AddTodoCommand) parser.parseCommand(TODOS_TAB_ID
```

Comment:
Should we change these constants to use the enums in TabSwitch? Since these are technically linked to the tabs name directly, any changes made there should affect these constants as well